assets/theme-css/sphinx-design/_dropdown.scss

```diff
@@ -5,10 +5,35 @@ details.sd-dropdown {
     font-weight: 700;
     // don't overlap the chevron
     padding-right: 3em !important;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    -webkit-user-select: none;
```

Comment:
My text editor warns that these attributes are non-standard and should be avoided, and `user-select` is baseline already. The same warnings occur in a few other places below, and I removed the browser-engine-specific styling there as well.